diff -urN hypermail-2.1.3.org/Makefile.in hypermail-2.1.3/Makefile.in
--- hypermail-2.1.3.org/Makefile.in	Fri Dec 14 21:12:59 2001
+++ hypermail-2.1.3/Makefile.in	Fri Dec 14 21:13:34 2001
@@ -28,9 +28,9 @@
  
 # Compiler to use
 CC=@CC@ 
-CFLAGS=@CFLAGS@ @INCLUDES@ -Ipcre $(WARNINGS)
+CFLAGS=@CFLAGS@ @INCLUDES@ $(WARNINGS)
 LIBS=@LIBS@
-LDFLAGS=@LDFLAGS@ -Lpcre
+LDFLAGS=@LDFLAGS@
 
 all:   hypermail support 
 
diff -urN hypermail-2.1.3.org/configure.in hypermail-2.1.3/configure.in
--- hypermail-2.1.3.org/configure.in	Fri Dec 14 21:12:59 2001
+++ hypermail-2.1.3/configure.in	Fri Dec 14 21:13:16 2001
@@ -11,8 +11,6 @@
 GDBM_INCLUDE=""
 GDBM_LIB=""
 
-AC_CONFIG_SUBDIRS(src/pcre)
-
 dnl ===========================================================================
 dnl Get host, target and build variables filled with appropriate info.
 dnl ===========================================================================
diff -urN hypermail-2.1.3.org/src/Makefile.in hypermail-2.1.3/src/Makefile.in
--- hypermail-2.1.3.org/src/Makefile.in	Fri Dec 14 21:12:59 2001
+++ hypermail-2.1.3/src/Makefile.in	Fri Dec 14 21:15:18 2001
@@ -27,10 +27,10 @@
 #WNOERROR=-Werror
 #WARNINGS=$(WNOERROR) -ansi -pedantic -Wall -Wtraditional -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Dlint
 
-CFLAGS=@CFLAGS@ @INCLUDES@ -Ipcre $(WARNINGS)
+CFLAGS=@CFLAGS@ @INCLUDES@ $(WARNINGS)
 YACC=@YACC@
 NETLIBS=@LIBS@
-LDFLAGS=@LDFLAGS@ -Lpcre
+LDFLAGS=@LDFLAGS@
 MISC_LIBS= -lm -lpcre
 OPT_LIBS=@EXTRA_LIBS@
 
@@ -54,7 +54,7 @@
 .c.o:
 	$(CC) -c $(CFLAGS) $<
 
-all:    pcre/libpcre.a hypermail$(SUFFIX) mail$(SUFFIX)
+all:    hypermail$(SUFFIX) mail$(SUFFIX)
 
 pcre/libpcre.a:
 	@cd pcre; $(MAKE) libtool libpcre.a CC="$(CC)" CFLAGS="$(CFLAGS)"
