diff -Nur hypermail-2.1.6.orig/Makefile.in hypermail-2.1.6/Makefile.in
--- hypermail-2.1.6.orig/Makefile.in	Mon Nov 25 06:36:44 2002
+++ hypermail-2.1.6/Makefile.in	Mon Feb 10 13:38:12 2003
@@ -28,9 +28,9 @@
  
 # Compiler to use
 CC=@CC@ 
-CFLAGS=@CFLAGS@ @INCLUDES@ -Ipcre $(WARNINGS)
+CFLAGS=@CFLAGS@ @INCLUDES@ $(WARNINGS)
 LIBS=@LIBS@
-LDFLAGS=@LDFLAGS@ -Lpcre
+LDFLAGS=@LDFLAGS@
 
 all:   hypermail support 
 
diff -Nur hypermail-2.1.6.orig/configure.in hypermail-2.1.6/configure.in
--- hypermail-2.1.6.orig/configure.in	Mon Nov 25 06:36:44 2002
+++ hypermail-2.1.6/configure.in	Mon Feb 10 13:38:00 2003
@@ -11,7 +11,7 @@
 GDBM_INCLUDE=""
 GDBM_LIB=""
 FNV_DEP=""
-SUBDIRS="src/pcre"
+SUBDIRS=""
 
 dnl ===========================================================================
 dnl Get host, target and build variables filled with appropriate info.
diff -Nur hypermail-2.1.6.orig/src/Makefile.in hypermail-2.1.6/src/Makefile.in
--- hypermail-2.1.6.orig/src/Makefile.in	Fri Jan 24 21:04:57 2003
+++ hypermail-2.1.6/src/Makefile.in	Mon Feb 10 13:38:41 2003
@@ -27,10 +27,10 @@
 #WNOERROR=-Werror
 #WARNINGS=$(WNOERROR) -ansi -pedantic -Wall -Wtraditional -Wshadow -Wpointer-arith -Wcast-qual -Wcast-align -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Dlint
 
-CFLAGS=@CFLAGS@ @INCLUDES@ -Ipcre $(WARNINGS)
+CFLAGS=@CFLAGS@ @INCLUDES@ $(WARNINGS)
 YACC=@YACC@
 NETLIBS=@LIBS@
-LDFLAGS=@LDFLAGS@ -Lpcre
+LDFLAGS=@LDFLAGS@
 MISC_LIBS= -lm -lpcre
 OPT_LIBS=@EXTRA_LIBS@
 
@@ -56,7 +56,7 @@
 .c.o:
 	$(CC) -c $(CFLAGS) $<
 
-all:    pcre/libpcre.a @FNV_DEP@ hypermail$(SUFFIX) mail$(SUFFIX) lang$(SUFFIX)
+all:    @FNV_DEP@ hypermail$(SUFFIX) mail$(SUFFIX) lang$(SUFFIX)
 
 pcre/libpcre.a:
 	@cd pcre; $(MAKE) libtool libpcre.a CC="$(CC)" CFLAGS="$(CFLAGS)"
